Sources/Fuzzilli/Mutators/OperationMutator.swift

```diff
@@ -117,6 +117,22 @@ public class OperationMutator: BaseInstructionMutator {
             newOp = BinaryOperation(chooseUniform(from: allBinaryOperators))
         case is ReassignWithBinop:
             newOp = ReassignWithBinop(chooseUniform(from: allBinaryOperators))
+        case let op as DestructArray:
+            var newIndices: [Int] = []
```

Comment:
I think I'd prefer if these did smaller changes. Also I'm not sure that this currently works? The number of outputs might now be different than the number of indices. How about something like:
```swift
var newIndices = Set(op.indices)
if newIndices.count > 0 {
    newIndices.remove(newIndices.randomElement()!)
    while newIndices.count != op.indices.count {
        newIndices.insert(Int.random(in: 0..<10))
    }
}
newOp = ... (newIndices.sorted(), ...)
...
```

Reply:
I think it's ok to use a larger maximum number (10) here than in the generator